db/migrate/20200311131527_drop_smart_variable_permissions.rb

```diff
@@ -0,0 +1,5 @@
+class DropSmartVariablePermissions < ActiveRecord::Migration[5.2]
+  def change
+    Permission.where(resource_type: 'VariableLookupKey').delete_all
```

Comment:
This needs to be `destroy_all` so that associated filterings (the connection table between roles and permission) are also removed